src/content/web/wasm.md

```diff
@@ -0,0 +1,74 @@
+---
+title: WebAssembly (Wasm) compilation
+description: Learn how to compile your Dart web app to WebAssembly.
+---
+
+The Dart team is excited to add
+[WebAssembly](https://webassembly.org/) as a compilation target when building
+Dart and [Flutter]({{site.flutter}}/wasm) applications for the web.
+
+Development of WebAssembly support remains ongoing,
+which will potentially result in frequent changes. 
+Revisit this page for the latest updates.
+
+:::note
+**Support for Wasm is now in beta!**
```

Comment:
@kevmoo how about we for Dart say that 1) it's in stable, but 2) it's "in preview, and still missing additional tooling changes to improve the end-to-end developer experience" ?

Reply:
I also added this text right before the steps:

> Wasm compilation is available in stable, but still in preview.
While we continue to optimize tooling for to improve developer experience,
you can try compiling Dart to Wasm today
by following the interim steps outlined here: